content/en/dora_metrics/setup/deployments.md

```diff
@@ -252,12 +274,13 @@ If the two metadata entries are defined for a service, only `extensions[datadogh
 
 ### Limitations
 
-- Change lead time stage breakdown metrics are only available for GitHub.
+- Change lead time stage breakdown metrics are only available for GitHub and GitLab.
+  - The GitLab integration is in Preview. Contact [Datadog support][8] to request access.
```

Comment:
There's nothing wrong with this line being here, but I might suggest taking it out. It's a little hard to see, so when the GitLab integration goes GA, it might be easy to miss for the person taking the preview notes out! I noticed there was no callout in the GitLab tab that contained the schema version too, so I think just one note is okay instead of having to worry about maintaining three.
```suggestion
```